hack/build/config.sh

```diff
@@ -35,7 +37,7 @@ function allPkgs {
     echo "$ret"
 }
 
-KUBERNETES_IMAGE="k8s-1.10.4@sha256:09ac918cc16f13a5d0af51d4c98e3e25cbf4f97b7b32fe18ec61b32f04ca1009"
+KUBERNETES_IMAGE="k8s-1.10.4@sha256:486064eddea289b17e150e6600fefc89dab9164d5cba07153c02888a35fed4f1"
```

Comment:
Incrementing the k8s version should probably be isolated in its own pr.

Reply:
yep, i submitted a separate pr for this. This will get resolve the next time i rebase this pr.